content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -227,8 +227,17 @@ kubectl describe pvc <pvc_name>
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
 
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. 
+The FlexVolume can be resized on pod restart as 'ExpandPersistentVolume' feature gate is enabled by default. 
```

Comment:
There is no need to say that `as 'ExpandPersistentVolume' feature gate is enabled by default. ` because it is already default.

Reply:
Removed the text about the feature gate.